includes/admin/views/html-admin-page-students-main.php

```diff
@@ -33,10 +45,32 @@
 				<?php $sensei_list_table->table_search_form(); ?>
 				<?php $sensei_list_table->display(); ?>
 			</form>
-			<?php do_action( 'sensei_learner_admin_extra' ); ?>
+	<?php
+	/**
+	 * Fires after rendering the students page container.
+	 * Allowing for additional content to be added.
+	 *
+	 * @hook sensei_learner_admin_wrapper_container
```

Comment:
```suggestion
	 * @hook sensei_learner_admin_extra
```

It has the same description as the next hook. Is that accurate?

Reply:
> It has the same description as the next hook. Is that accurate?

Basically, yes. There is a difference that is reflected in the description for this hook. It's designed to put an extra content here.

The other one is for wrapping content, and it has a counterpart above.

(Thanks for the suggestion with the fix!)